Gems/ScriptCanvas/Code/Editor/Components/EditorGraph.cpp

```diff
@@ -1418,6 +1418,7 @@ namespace ScriptCanvasEditor
             {
                 data->m_scriptCanvasEntity.reset(entity);
                 graph->MarkOwnership(*data);
+                graph->MarkVersion();
```

Comment:
This was requested by @carlitosan during implementation of the ScriptCanvas asset creator.